packages/sdk/src/protocols/filter.ts

```diff
@@ -294,7 +294,21 @@ class FilterSDK extends BaseProtocolSDK implements IFilterSDK {
     decoders: IDecoder<T> | IDecoder<T>[],
     callback: Callback<T>
   ): Promise<Unsubscribe> {
-    const subscription = await this.createSubscription();
+    const pubsubTopics = this.getPubsubTopics(decoders);
```

Comment:
Are we only changing the `subscribe` function on Filter?

IMO - we should also update the `createSubscription` function to not use the `DEFAULT_PUBSUB_TOPIC` and instead fallback on TWN cluster.

The `subscribe` function within `Filter` is not supposed to be directly used and only satisfies the `IReceiver` API.

Reply:
> we should also update the createSubscription

I agree and will push some changes that I mentioned here https://github.com/waku-org/js-waku/issues/1967#issuecomment-2071045829

Reply:
should be addressed with recent changes